### PR TITLE
Fix bfe stauanlagen pic links

### PR DESCRIPTION
**Pull request by Sgachet**

### CODE REVIEW

**Comment by AFoletti** [comment]
Could you please provide a testlink?

**Comment by AFoletti** [comment]
Hm I don't see any picture.
[This feature](https://mf-chsdi3.dev.bgdi.ch/ltgas/rest/services/ech/MapServer/ch.bfe.stauanlagen-bundesaufsicht/130676842/extendedHtmlPopup?lang=fr) should have one [see](http://www.uvek-gis.admin.ch/BFE/bilder/ch.bfe.stauanlagen-bundesaufsicht/130675954.jpg)

**Comment by Sgachet** [comment]
Sorry, the test link was wrong... 

Now, it's work !

**Comment by AFoletti** [comment]
Yeah, now it works 😄 
LGTM then, but I'll let an official merger merge your code

File: chsdi/templates/htmlpopup/stauanlagenbundesaufsicht.mako [hunk 1]
```diff
@@ -24,77 +24,32 @@
...
+        <tr><th class="cell-left">${_('tt_ch.bfe.stauanlagen-bundesaufsicht_crestlevel')} [${_('abk_meter_ueber_meer')}]</th>         <td>${c['attributes']['crestlevel']}</td></tr>
+        <tr><th class="cell-left">${_('tt_ch.bfe.stauanlagen-bundesaufsicht_crestlength')} [m]</th>                                   <td>${c['attributes']['crestlength']}</td></tr>
+        <tr><th class="cell-left">${_('tt_ch.bfe.stauanlagen-bundesaufsicht_damtype')}</th>                                           <td>${c['attributes'][damtype]}</td></tr>
     </table>
     % if image_exist:
     <div class="thumbnail-container">
         <div class="thumbnail">
-            <a href="${img_url}" target="_blank"><img class="image" src="${img_url}" /></a>
+             <a href="${img_url}" target="_blank">
+             <img class="image" src="${img_url}" alt=""/>
```
Comment (oterral) [inline chsdi/templates/htmlpopup/stauanlagenbundesaufsicht.mako]: a closing `</a>` is missing

File: chsdi/templates/htmlpopup/stauanlagenbundesaufsicht.mako [hunk 2]
```diff
@@ -24,77 +24,32 @@
...
-        </tr>
-        <tr>
-            <th class="cell-left">${_('tt_ch.bfe.stauanlagen-bundesaufsicht_crestlength')} [m]</th>
-            <td>${c['attributes']['crestlength']}</td>
-        </tr>
-        <tr>
-            <th class="cell-left">${_('tt_ch.bfe.stauanlagen-bundesaufsicht_damtype')}</th>
-            <td>${c['attributes'][damtype]}</td>
-        </tr>
+        <tr><th class="cell-left">${_('tt_ch.bfe.stauanlagen-bundesaufsicht_stauanlage')}</th>                                        <td>${c['attributes']['facilityname']}</td></tr>
```
Comment (oterral) [inline chsdi/templates/htmlpopup/stauanlagenbundesaufsicht.mako]: Why removing the indentations?

**Comment by Sgachet** [comment]
Thanks @oterral for the review,
Changes are done

File: chsdi/templates/htmlpopup/stauanlagenbundesaufsicht.mako [hunk 3]
```diff
@@ -24,7 +24,7 @@
 
         from urllib2 import urlopen
         img_url = "http://www.uvek-gis.admin.ch/BFE/bilder/ch.bfe.stauanlagen-bundesaufsicht/"+str(c['attributes']['facility_stabil_id'])+".jpg"
```
Comment (oterral) [inline chsdi/templates/htmlpopup/stauanlagenbundesaufsicht.mako]: Is `https` working ?

**Comment by Sgachet** [comment]
@oterral @loicgasser i'm not really sure about what I do... but seem's to work

**Comment by AFoletti** [comment]
Could we not make it protocol-agnostic as suggested by Martin himself?

**Pull request merged**